filesystem/mountpoint.go

```diff
@@ -123,20 +128,85 @@ func parseMountInfoLine(line string) *Mount {
 	if err != nil {
 		return nil
 	}
+	mnt.Subtree = unescapeString(fields[3])
```

Comment:
We could then just have a `BindMount` boolean field instead of a `Subtree` field, set like:
```go
mnt.BindMount = fields[3] != "/"
```